provisioner/env.go

```diff
@@ -77,3 +83,8 @@ func getOpenEBSServiceAccountName() string {
 func getNFSServerImage() string {
 	return menv.GetOrDefault(NFSServerImageKey, string(NFSServerDefaultImage))
 }
+
+// getEnv fetches the provided environment variable's value
+func getEnv(envKey string) (value string) {
```

Comment:
no need to add a new function. we can use `menv.Get` instead

Reply:
updated PTAL